keyboards/xiaomi/mk02/config.h

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2015 Jun Wako <wakojun@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x0483
+#define PRODUCT_ID      0x5710
```

Comment:
This vendor ID is already in use, by STM, as a joystick controller. 

It may be best to use a set, that isn't in use.

Reply:
This is the vid/pid pair used by the stock firmware; I wasn't sure if I should keep it or generate a unique one.

Reply:
Xiaomi shouldn't have used it in the first place, by all accords. So, I would prefer that the VID/PID be changed.  

It's an officially registered ID that doesn't belong to this keyboard (or vendor), so I'd feel more comfortable if it were changed. 

If another member has an opinion about it, I'll defer to them, but "because it was already that way" isn't a good reason to continue to use a VID/PID assigned to a different vendor, IMO.

Reply:
I agree with @drashna here.